### ISSUE TITLE
User report - account-level rate limiter

### ISSUE THREAD

**Issue by ttodua** [ccxt/ccxt#19772]
### Preliminary Checks

- [X] I have already searched for [existing issues](https://github.com/ccxt/ccxt/issues) and confirmed that this issue is not a duplicate

### Is your feature request related to a problem? Please describe

> Lately my bots are hitting API limits. They run on different IPs (and different servers) but apparently with ByBit this is grouped on a UID level. What's the recommended way to mitigate this? I can't run all my bots under a single ccxt instance for performance reasons.
> I suppose I could write a ccxt proxy to consolidate the rate limiter but that seems overly complex

### Describe the solution you'd like

n/a

**Comment by ttodua** [comment]
@snailwhale
yes, unfortunately at this stage you have to handle it on your own. in somewhat foreseeable future we might publish some exemplary snippet to help users to setup such proxy, but at this stage you have to handle that yourself

**Comment by samgermain** [comment]
One solution would be to multiply the `exchange.rateLimit` value by the number of bot instances you have

```python
import ccxt
exchange = ccxt.bybit()
exchange.rateLimit = exchange.rateLimit * numberOfBots
```

this of course has some cons, such that if you have one bot doing 90% of what you want, then it will only be able to send requests at a rate of rateLimit/numberOfBots

**Comment by Hadevmin** [comment]
Hi,

I don't remember since what version of ccxt, but recently I often get the exception: 
`RateLimitExceeded : bybit {"retCode":10006,"retMsg":"Too many visits. Exceeded the API Rate Limit.","result":{},"retExtInfo":{},"time":1699404365845}`
Whereas I haven't modified the behavior of my algorithm for Bybit.
Previously, it rarely came up when I was writing bad code.

**Comment by javeoff** [comment]
did you find any solutions to solve it?
